dev-util/cmake/cmake-3.19.2.ebuild

```diff
@@ -18,14 +18,15 @@ LICENSE="CMake"
 SLOT="0"
 [[ "${PV}" = *_rc* ]] || \
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x64-solaris ~x86-solaris"
-IUSE="doc emacs ncurses qt5 test"
+# system-jsoncpp is only meant to be disabled during prefix bootstrap
+IUSE="doc emacs ncurses qt5 +system-jsoncpp test"
```

Comment:
We should probably force it in profiles then?

Reply:
ok. I can add that to the profiles.
I believe that means I should force it on in `profiles/base/package.use.force` and then drop the force in `profiles/prefix/darwin/macos/package.use.force`, right?

Reply:
Hmm. If I add `dev-util/cmake -system-jsoncpp` to `darwin/macos/package.use.force` that would mean it would force it to be disabled even after bootstrap. Is there a way to force it on, and then just disable the forced useflag for a subsequent profile?

Reply:
So, I don't think there's a good way to force this useflag in the profiles. I think the default `+system-jsoncpp` should be sufficient. If needed, we can add a warning to the useflag description saying it is for prefix bootstrap use only.

Reply:
K. I added the profile changes.